### ISSUE TITLE
Misc: Ignore "no audio" & "no subtitle" track

### ISSUE THREAD

**Issue by Cananbolt** [stax76/mpv-scripts#9]
Currently we have, cycle audio and subtitle tracks, include only languages you know. Is it possible to add onto this and get rid of the "no audio" & "no subtitle" track (which is labeled as the 0 track)?

**Comment by stax76** [comment]
I added some options, the code is complex, I hope it's not buggy, please help to test common use cases!

**Comment by Cananbolt** [comment]
Before I did anything I switched...
Line 148
include_no_sub = true 
To
include_no_sub = false
(Just my personal preference)

After that I did an initial testing. 

I had a video with only 1 sub track and 1 audio track and when I used the keybind it did not go to the "no track" item. Exactly what I wanted.

Another video with 2 audio tracks, both with the languages I chose in the script. Pressed the keybind and it switched to them as intended while not choosing the "no track" item in between. In that same video, it has 36 sub tracks. Filled with different languages. I only wanted 1 that being English and, as intended it only switched to the English sub tracks and it did not select the "no track" item. Exactly how I wanted.

Now I did a test with...

(Line 147)
include_no_sub = true
(Line 148)
include_no_sub = true

It included the no tracks for both the audio tracks and the sub tracks as intended based on the config above.

So far I'm not experiencing any hiccups or bugs. This feature along with selecting only the languages you only want is fantastic. Thanks so much for sharing the script. Feel free to close the issue whenever you like, this is exactly what I wanted.
